### ISSUE TITLE
Virtual keyboard: attribute :mask(VaInput) does not work

### ISSUE THREAD

**Issue by EugenGedroyc** [epicmaxco/vuestic-ui#3862]
**Is your feature request related to a problem? Please describe.**

I'm add [virtual keyboard](https://virtual-keyboard.js.org/vuejs/)
but it does not work with attribute :mask

```
 <va-input
    v-model="transportNumber"
    clearable
    :mask="{ blocks: [1, 3, 2] }"
/>
```

If typing characters with **click** event, then mask does not work (the same behavior VaInput without :mask).  
If typing characters with **touch** event, then only first character appears in VaInput, the rest disappear, and if you click in this VaInput, then the rest of the characters appear at once. But mask also does not work: with click event same text in VaInput. 

**Describe the solution you'd like**

Virtual keyboard change value of Input in direct. But :mask probably work with keyup\keydown events ?

**Describe alternatives you've considered**

Where to look for ways to solve the problem? 

**Additional context**

similar issue: #3861

**Comment by m0ksem** [comment]
Yeah. We use cleave.js to handle masks. I think it only listens for keyboard events. When I was making a virtual keyboard, I emitted `new KeyboardEvent`.

**Comment by m0ksem** [comment]
Not sure if it works with new useInputMask composable.

**Comment by EugenGedroyc** [comment]
> Yeah. We use cleave.js to handle masks. I think it only listens for keyboard events. When I was making a virtual keyboard, I emitted `new KeyboardEvent`.

I wonder how it was possible to do this for the inputs, if KeyboardEvent is disallowed for [security reasons](https://stackoverflow.com/questions/45457875/programmatically-send-keys-to-input-using-dispatchevent)
